code/modules/admin/admin.dm

```diff
@@ -427,7 +427,7 @@
 	if(confirm == "Yes")
 		SSticker.delay_end = 0
 		SSblackbox.add_details("admin_verb","Reboot World") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
-		world.Reboot("Initiated by [usr.client.holder.fakekey ? "Admin" : usr.key].", "end_error", "admin reboot - by [usr.key] [usr.client.holder.fakekey ? "(stealth)" : ""]", 10)

```

Comment:
You could change the confirm from "restart the round?" to a 3 button prompt for what kind of restart